zap-pipeline fails on 'Publish ZAP Report to SonarQube ...' step

zap-pipeline fails on 'Publish ZAP Report to SonarQube ...' step with the following error;
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':sonarqube'.
> Could not resolve all files for configuration ':testCompileClasspath'.
   > Could not resolve junit:junit:4.12.
     Required by:
         project :
      > Could not resolve junit:junit:4.12.
         > Could not get resource 'http://repo1.maven.org/maven2/junit/junit/4.12/junit-4.12.pom'.
            > Could not GET 'http://repo1.maven.org/maven2/junit/junit/4.12/junit-4.12.pom'. Received status code 501 from server: HTTPS Required
```

--- a/sonar-runner/build.gradle
+++ b/sonar-runner/build.gradle
@@ -64,7 +64,7 @@ dependencies {
 repositories {
   repositories {
     maven {
-      url "http://repo1.maven.org/maven2/"
+      url "https://repo1.maven.org/maven2/"
     }
     maven {
       url "https://plugins.gradle.org/m2/"

